akvo/rsr/models/country.py

```diff
@@ -85,7 +85,7 @@ def iati_country(self):
         return codelist_value(codelist_models.Country, self, 'country')
 
     def iati_country_unicode(self):
-        return str(self.iati_country())
+        return unicode(self.iati_country())
```

Comment:
Do we really need unicode() here? After all self.iati_country() returns the codelists.models.Country.name field. Should be usable as is, no?